guides/line-management/probation.md

```diff
@@ -0,0 +1,87 @@
+# Probation at Made Tech
+
+When someone joins Made Tech, they're "on probation" for a short time.
+This means their notice period is shorter - usually one week.
+
+At the end of probation, people either pass or fail. Most people pass.
+Rarely, you'll extend probation, or pass someone early.
+
+This guidance is written for line managers, but may be useful for people on probation too.
+
+- [What happens during probation?](#what-happens-during-probation)
+- [Explaining how to pass probation](#explaining-how-to-pass-probation)
+- [One-to-one meetings during probation](#one-to-one-meetings-during-probation)
+- [Finishing probation](#finishing-probation)
+
+## What happens during probation?
+
+In the first week, explain what probation is, and how to pass it.
+
+Each week, meet your direct one-to-one.
+Get feedback from their team, and reflect on it together.
+Talk often about whether they're on-track to pass.
+
+At the end of probation, pass or fail it, and explain what happens next.
+
+## Explaining how to pass probation
+
+Talk about what good looks like for their role. [The handbook role pages](../../roles/README.md) may help with this.
+It's good to spend the first two weeks talking about this.
+
+Explain that feedback is useful evidence on how they're performing.
+
+Tell your direct to ask their team for feedback. You can talk to your direct's team to help get more feedback, if needed.
+
+### Worries about failing probation
+
+It can be stressful to be on probation. Explain early that:
```

Comment:
Maybe also re-iterate that probation is also useful for the individual to determine if working here is right for them, and also that extending probation because there's a need to see some competencies that aren't easily showcased _right now_ is relatively common and not a negative reflection

Reply:
I agree with all of this - but I'm concerned about adding prose here. I'm trying to address two things:
- I'm responsible for welfare, and I want to highlight that people worry about probation
- I'm responsible for line management, and I have someone worried about probation - what should I do?

More broadly: I think communities/discussion are a better place for the more nuanced points, not this guidance.

(side note: My understanding is that extending probation is relatively rare (and this document describes it as such) - @yaseminercan might have better data on that if it's in doubt.)

Reply:
>and I have someone worried about probation - what should I do?
I think explaining the way probation also benefits the employee is part of that reassurance, I'd consider that something a line manager should highlight. I'm ok with the nuance of that being part of discussions rather than guidance though

Reply:
I am coming round to this being a good thing to include, but I haven't found a good one-line expression of it. I'll think about it and we can hopefully iterate on this page further.